### PR TITLE
firefox: add page

### PR DESCRIPTION
**Pull request by mxmxyz**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/firefox.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# firefox
+
+> A free and open source web browser.
+> More information at https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options.
+
+- Open firefox at www.duckduckgo.com:
+
+`firefox http://www.duckduckgo.com`
```
Comment (agnivade) [inline pages/common/firefox.md]: http://www.duckduckgo.com should be inside tokens (`{{}}`)

File: pages/common/firefox.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open it at a new window:
+
+`firefox -new-window http://www.duckduckgo.com`
```
Comment (agnivade) [inline pages/common/firefox.md]: Same as above

File: pages/common/firefox.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Go incognito:
+
+`firefox -private-window`
+
+- Search 'wikipedia' on your default search engine:
+
+`firefox -search wikipedia`
```
Comment (agnivade) [inline pages/common/firefox.md]: Same for `wikipedia`

File: pages/common/firefox.md (hunk 1 above)
Comment (pxgamer) [reply pages/common/firefox.md]: Can we also use HTTPS? 🤔

File: pages/common/firefox.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open firefox at www.duckduckgo.com:
+
+`firefox {{http://www.duckduckgo.com}}`
+
+- Open it at a new window:
+
+`firefox -new-window {{http://www.duckduckgo.com}}`
+
+- Go incognito:
```
Comment (pxgamer) [inline pages/common/firefox.md]: Maybe this would read better?
```suggestion
- Open an incognito window:
```

File: pages/common/firefox.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`firefox -private-window`
+
+- Search 'wikipedia' on your default search engine:
+
+`firefox -search {{wikipedia}}`
+
+- Launch firefox with no extensions:
```
Comment (pxgamer) [inline pages/common/firefox.md]:
```suggestion
- Launch firefox without extensions:
```

**Pull request merged**